### DISCUSSION TITLE
"Read later" functionality?

### DISCUSSION THREAD

**Discussion by v-marinkov** [beromir/Servas#85] [General]
Can Servas be tailored as a read-it-later substitute to Wallabag or Pocket? What is needed is the ability to mark as "read/unread" and to archive. Potentially a built-in reader mode could be useful.
The main problem with Wallabag is the 9-year-old request to allow filtering by more than one tag...

**Comment by beromir** [comment]
Hey,
it is not possible to mark links as read/unread, nor is there an archive. You could add these functionalities through tags and groups, but I doubt that is what you had in mind. I don't see Servas as a read-later service, more as a place to save links and organize them. I may add the archive function in the future.
There are many great apps out there, which could better suit your needs.

**Reply by v-marinkov** [reply]
Thanks for confirming!
